stubs/PyAutoGUI/pyautogui/__init__.pyi

```diff
@@ -72,10 +212,10 @@ def size() -> Size: ...
 @overload
 def onScreen(x: tuple[float, float], y: None = ...) -> bool: ...
 @overload
-def onScreen(x: float, y: float) -> bool: ...
+def onScreen(x: _NormalizeableXArg | None, y: SupportsInt | None) -> bool: ...
```

Comment:
Hmm, not sure it makes so much sense to use the `_NormalizeableXArg` alias here, since in this case, this is an overloaded function, and this is the second overload. If both `x` and `y` are provided, and `y` is not `None`, then `x` cannot be a `str` or a `Sequence`.

I would just use `SupportsInt` for both arguments here, for the second overload.

Reply:
Looking back at this. This overload just seems wrong.
First it should be more flexible than tuples, even the docstring suggest it's meant to be useable with a list.
Second, it's directly calling `_normalizeXYArgs`, so why not just do like everywhere else and just have `x: _NormalizeableXArg | None, y: SupportsInt | None` ?

Only thing is, the docstring doesn't suggest this is meant to be used to find whether a window is on screen (by passing a string as the first arg), but I could totally see users do that.

Due to this incertainty, I've asked the question on pyautogui, hopefully this'll get an answer: https://github.com/asweigart/pyautogui/issues/729

In the mean time, I'd heir on the side of caution (and simplicity), following current implementation. And just do:
`def onScreen(x: _NormalizeableXArg | None, y: SupportsInt | None) -> bool: ...` without any overload.

Reply:
The new signature you've given it looks fine to me -- we just need to make the `y` argument optional:

```suggestion
def onScreen(x: _NormalizeableXArg | None, y: SupportsInt | None = ...) -> bool: ...
```